src/promptflow-devkit/promptflow/_sdk/_orchestrator/experiment_orchestrator.py

```diff
@@ -83,6 +83,47 @@ def __init__(self, client, experiment: Experiment = None):
         self._node_runs = {}
 
     def test(
+        self,
+        template: ExperimentTemplate,
+        inputs=None,
+        environment_variables=None,
+        **kwargs,
+    ):
+        """Test experiment.
+
+        :param template: Experiment template to test.
+        :type template: ~promptflow.entities.ExperimentTemplate
+        :param inputs: Input parameters for experiment.
+        :type inputs: dict
+        :param environment_variables: Environment variables for flow.
```

Comment:
flow -> experiment

Reply:
Will refine the env var part in the next pr.